values.yaml

```diff
@@ -266,5 +266,14 @@ weblate:
         paths:
           - path: /(weblate)(.*)
             pathType: Prefix
+    apiUrl:
```

Comment:
Same here:
```
api:
    url: "https://onlineberatung.net/weblate"
    key: ""
```

Reply:
And it is wrongly intended i think.
should be inside weblate. Looks like its inside ingress?

Reply:
yes, this is fixed